app/src/main/graphql/project.graphql

```diff
@@ -1,5 +1,5 @@
-query FetchProjects($first: Int = 15, $cursor: String, $sort: ProjectSort, $backed: Boolean, $recommended: Boolean, $categoryId: String, $starred: Boolean, $staffPicks: Boolean, $searchTerm: String) {
-  projects(first: $first, after:$cursor, sort:$sort, backed:$backed, recommended:$recommended, categoryId: $categoryId, starred:$starred, staffPicks: $staffPicks, term: $searchTerm) {
+query FetchProjects($first: Int = 15, $cursor: String, $sort: ProjectSort, $state: PublicProjectState,  $backed: Boolean, $recommended: Boolean, $categoryId: String, $starred: Boolean, $staffPicks: Boolean, $searchTerm: String) {
```

Comment:
Had to update the query and graphQL schema to have `PublicProjectState` definition available.